gradle.properties

```diff
@@ -13,10 +13,10 @@ cordaProductVersion = 5.0.0-INTEROP
 ## IMPORTANT:
 ## The interop feature branches track api revisions separately to the mainline branch.
 ## API version of last merge from corda mainline: 755
```

Comment:
Is this line up to date? Still 755 on mainline?

Reply:
yes, at that time it was 755 still